ebean-api/src/main/java/io/ebean/StdOperators.java

```diff
@@ -0,0 +1,198 @@
+package io.ebean;
+
+import io.ebean.Query.Property;
+
+import java.util.Collection;
+
+@Deprecated(since = "experimental")
+public final class StdOperators {
+
+  // ---- Functions ---- //
+
+  public static Property<Number> sum(Property<? extends Number> property) {
+    return Property.of("sum(" + property + ")");
+  }
+
+  public static Property<Number> count(Property<?> property) {
+    return Property.of("count(" + property + ")");
+  }
+
+  public static <T> Property<T> avg(Property<T> property) {
+    return Property.of("avg(" + property + ")");
+  }
+
+  public static <T> Property<T> max(Property<T> property) {
+    return Property.of("max(" + property + ")");
+  }
+
+  public static <T> Property<T> min(Property<T> property) {
+    return Property.of("min(" + property + ")");
+  }
+
+  public static <T> Property<T> coalesce(Property<T> property, Object value) {
+    return Property.of("coalesce(" + property.toString() + "," + sqlValue(value) + ")");
+  }
+
+  public static Property<String> lower(Property<String> property) {
+    return Property.of("lower(" + property + ")");
+  }
+
+  public static Property<String> upper(Property<String> property) {
+    return Property.of("upper(" + property + ")");
+  }
+
+  public static Property<String> concat(Property<?> property, Object... values) {
+    StringBuilder expression = new StringBuilder(50);
+    expression.append("concat(").append(property.toString());
+    for (Object value : values) {
+      expression.append(",").append(sqlConcatString(value));
+    }
+    expression.append(")");
+    return Property.of(expression.toString());
+  }
+
+  private static String sqlConcatString(Object value) {
```

Comment:
Use sqlValue instead?

Reply:
I don't think so no unless we leave it up to the database to cast numbers to strings? I'll adjust this for sql injection.